tests/testthat/test-linear_pool.R

```diff
@@ -286,7 +299,7 @@ test_that("(weighted) quantiles correctly calculated", {
                tolerance = 1e-3)
   expect_equal(weighted_quantile_expected,
                as.data.frame(weighted_quantile_actual),
-               tolerance = 1e-3)
+               tolerance = 1e-2)
```

Comment:
my hope would be that after going to the more precise/exact/deterministic weighted quantile computation outlined in an above comment, we could shift back to the more precise expected tolerance here